common/script/fns/randomDrop.js

```diff
@@ -69,7 +69,7 @@ module.exports = function(user, modifiers, req) {
         base2[name2] = 0;
       }
       user.items.hatchingPotions[drop.key]++;
-      drop.type = 'HatchingPotion';
```

Comment:
Yes, it's not a typo. @a-ayyash It's a key that is used in other parts of the code to control behaviour depending on what type of thing is dropped. It's deliberately written as a single word because that's common in programming for keys.

Generally in Habitica (with some exceptions that are getting rarer), all user-visible text is defined in files under the `locales` directory. Any text elsewhere is likely to be program code and usually shouldn't be modified when it seems like a typo.